addons/website/views/snippets/s_cta_card.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_card" name="Card Call to Action">
+    <section class="s_cta_card o_cc o_cc2 pt64 pb64" data-oe-shape-data="{'shape':'web_editor/Blocks/03'}">
+        <div class="o_we_shape o_web_editor_Blocks_03"/>
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-lg-9">
+                    <h2>50,000+ companies run Odoo to grow their businesses.</h2>
```

Comment:
Little workaround to correctly style demo-text prior to user customizations 😉
The user will remove it by just typing 👍

```suggestion
                    <h2>50,000+ companies run Odoo<br class="d-none d-lg-inline"> to grow their businesses.</h2>
```
| before | after |
|--------|--------|
| <img width="1375" alt="image" src="https://github.com/user-attachments/assets/8ddb6a81-fa85-4120-bb25-392e7bee0b69"> | <img width="1373" alt="image" src="https://github.com/user-attachments/assets/1a8cc2f7-de04-4073-8e20-f688b717dccb"> |